CHANGELOG.md

```diff
@@ -7,9 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-
-## [4.0.0-RC1] - 2023-11-21
```

Comment:
This is intentional - the `4.0.0-RC1` release is stuck in a limbo because of a Maven Central outage. Eventually, I want to coalesce all release candidate changelog entries into the main `4.0.0` release anyway.